ArduPlane/quadplane.cpp

```diff
@@ -3533,10 +3533,10 @@ float QuadPlane::forward_throttle_pct()
             return 0;
         } else {
             // calculate fwd throttle demand from manual input
-            float fwd_thr = rc_fwd_thr_ch->percent_input();
+            float fwd_thr = rc_fwd_thr_ch->norm_input() * 100.0f;
 
-            // set forward throttle to fwd_thr_max * (manual input + mix): range [0,100]
-            fwd_thr *= .01f * constrain_float(fwd_thr_max, 0, 100);
+            // set forward throttle to fwd_thr_max * (manual input + mix): range [-100,100]
+            fwd_thr *= .01f * constrain_float(fwd_thr_max, -100, 100);
```

Comment:
I don't think you need these changes. Although we might need a new param for `fwd_thr_min` which would default to 0 for current behavior.